torch/_inductor/compile_fx.py

```diff
@@ -164,7 +164,8 @@ def compile_fx_inner(
                 elif complex_memory_overlap_inputs:
                     log.warning("skipping cudagraphs due to complex input striding")
 
-    result = align_inputs(compiled_fn, example_inputs, range(num_fixed))
+    # result = align_inputs(compiled_fn, example_inputs, range(num_fixed))
```

Comment:
nit rm

Reply:
This is a hack.

Reply:
From chat: Natalia and I talked about it and it sounds like we have to do some sort of TensorIterator style allow for all of the different alignments and compile each one and dispatch to the right one for the input (hopefully usually hitting 0)

cc @ngimel